WebAppConfig.ps1

```diff
@@ -0,0 +1,67 @@
+param(
+	[string]$webAppTarget,
+	[string]$webAppHosts,
+	[string]$renewXNumberOfDaysBeforeExpiration = "-1", # set this to e.g. 65 after first request to avoid hitting the cert-limit if the job is triggerd manually more than 5 times
+	[string]$tenantId = "<default tenant Id>",
+	[string]$subscriptionId = "<default subscription Id>",
+	[string]$webAppLetsEncrypt = "<default letsencrypt web app>",
+	[string]$resourceGroupLetsEncrypt = "<default resource group of letsencrypt web app>",
+	[string]$clientId = "<default client Id>",
+	[string]$clientSecret = "<default client secret>",
```

Comment:
This is probably not something you want to check in with the default value, I would think this should get passed in the command-line every time

Reply:
I do not (completele) agree here. Yes, these values should not be checked in, but I download the script, enter those values and save the modified script as "WebAppConfig_NAME.ps1". Then I can re-execute this script each time I add an app to NAME. I maintain several AppServices in different directories so this is the most comfortable way for me. Of course you do not need to adapt/set this default values and can always provide them via a parameter, but I do not want to force that...